Automattic/newspack-popups
# Make settings more consumable by other plugins

### All Submissions:






### Changes proposed in this Pull Request:



In order to enable managing plugin's settings in the main Newspack plugin (https://github.com/Automattic/newspack-plugin/pull/651), these changes have to made.

### How to test the changes in this Pull Request:

1. Open the plugin's settings
2. Change the settings, verify the options are updated (`suppress_newsletter_campaigns`, `suppress_all_newsletter_campaigns_if_one_dismissed`, `suppress_donation_campaigns_if_donor`, `newspack_newsletters_non_interative_mode`)

### Other information:
